mxcube3/routes/login.py

```diff
@@ -46,6 +46,8 @@ def login():
             logging.getLogger("MX3.HWR").info(msg)
             res = deny_access(str(ex))
 
+        session.permanent = True
```

Comment:
just a comment. Without this line, the cookie would be valid only during the "Session", and deleted on browser closing. This causes a bug where the local storage says logged in but the absence of the cookie says the opposite, it makes the loading break. In order to reproduce (without this line), just login as normal, close the browser and go to the main page again.

The [requireAuth()](https://github.com/mxcube/mxcubeweb/blob/develop/ui/src/index.js#L42) returns before the _getLoginInfo_ finishes. This still needs to be addressed, but as long as the cookie is valid it _should_ be ok 🤔

Reply:
Aha, I wondering if we should not use this instead: https://flask-login.readthedocs.io/en/latest/#cookie-settings (REMEMBER-COOKIE)

Reply:
we should... but I did not manage to have those working 🤪...

Reply:
Alright :), its a good start. We can have a look together once you have the behavior you like :)